internal/assets/manager.go

```diff
@@ -81,10 +80,7 @@ func (am *assetManager) CreateTokenPoolWithID(ctx context.Context, ns string, id
 		return nil, err
 	}
 
-	if pool.Author == "" {
-		pool.Author = config.GetString(config.OrgIdentity)
-	}
-	author, err := am.identity.Resolve(ctx, pool.Author)
+	err := am.identity.ResolveInputIdentity(ctx, &pool.Identity)
```

Comment:
I noted this in the identity code below too, but feels like we're ultimately going to need to support 2 "author" identities here (1 for the token pool connector and 1 for the blockchain). They're likely to be the same in simple cases (ie Ethereum+ERC1155), but we'll need to allow them to be specified differently in the input payload if needed.

Reply:
Agreed. Let's accept that this PR leaves things in a "still broken" state before+after for tokens, and address in #215 after merge.